addons/binding/org.openhab.binding.konnected/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+	http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:konnected:module">
+		<parameter-group name="actions">
+			<label>Actions</label>
+			<description></description>
+		</parameter-group>
+
+		<parameter name="authToken" type="text" required="true">
+			<label>Auth_Token</label>
+			<description>This is the Authority Token used to communicate with the Konnected Module.</description>
+		</parameter>
+		<parameter name="hostAddress" type="text">
```

Comment:
so the parameter is optional you are correct i have the config parameter because in my early testing i was having problems with the service grabbing the address of the server so the parameter allowed me to bypass this and manually define it quickly.  However, assuming i have now coded this properly with the network service and the callback address now always comes from the OSGI property which the user can be directed to configure in the case of complicated network situations then i would agree that it is no longer needed here and can be removed.

Reply:
removed .... this simplified some of the code as well